spec/lang/lexer/long_comment_spec.lua

```diff
@@ -6,6 +6,30 @@ local function string_trim(s)
    return (s:gsub("^%s*(.-)%s*$", "%1"))
 end
 
+--- removes leading whitespace from every line of a multiline string
+local function dedent(s)
```

Comment:
Instead of copying them around, you can add those utility functions to spec/util.lua, then localize them at the top with `local dedent = util.dedent` for brevity.